packages/builder/src/stores/builder/automations.ts

```diff
@@ -1134,7 +1138,7 @@ const automationActions = (store: AutomationStore) => ({
    * @returns
    */
   shiftBranch: (pathTo: Array<any>, block: AutomationStep, direction = -1) => {
```

Comment:
Should the block input not be `BranchStep` instead?

Reply:
Good call! Much nicer.